reference/hoon-expressions/rune/buc.md

```diff
@@ -22,10 +21,10 @@ x)))`, it's a normalizer and can be used as a mold.
 statically, so we can't actually tell if a mold matches this
 definition perfectly.  This is not actually a problem.)
 
-Validation, though very
-important, is a rare use case.  Except for direct raw input,
-it's generally a faux pas to rectify nouns at runtime -- or even
-in userspace.
+Validation (done with [`$|`](#bucbar), though very important, is a rare use
+case. Except for direct raw input, it's generally a faux pas to rectify nouns at runtime -- or even
+in userspace. Nonetheless they are sometimes utilized for structures that will
+be faster to use if they satisfy some validating gate.
```

Comment:
I think these normative claims are too strong. And the word "rectify" isn't quite right. `$|` is used to add validation predicates that the type system cannot itself enforce. It does not affect bunting, casting, or type inference. And it doesn't change the product of a mold-gate. It simply applies additional validation when clamming with a mold.

Reply:
Yeah I didn't know what was meant by "rectify" here either so I just left most of it as it was.

Should I just nix everything that's there and replace it with a paraphrase of what you wrote here? I'm not sure which specific sentences you think are too strong.

Reply:
I think doing what you suggest makes sense here @syzygyzer

Reply:
@syzygyzer do you have the time to make this change this week?